setup.cfg

```diff
@@ -29,3 +29,9 @@ process-timeout=300
 #   this rule.
 ignore=E501,E265,W602
 exclude=migrations,.git,.pycharm_helpers,test_root/staticfiles,node_modules
+
+[isort]
+indent='    '
+line_length=120
+multi_line_output=3
+skip=envs,migrations
```

Comment:
Just a reminder to update the cookie-cutter with the same, if you haven't already.  Thanks.